pkg/configtx/example_test.go

```diff
@@ -1203,579 +476,15 @@ func ExampleNewCreateChannelTx() {
-	// 					},
-	// 					"ChannelRestrictions": {
-	// 						"mod_policy": "Admins",
-	// 						"value": {
-	// 							"max_count": "1"
-	// 						},
-	// 						"version": "0"
-	// 					},
-	// 					"ConsensusType": {
-	// 						"mod_policy": "Admins",
-	// 						"value": {
-	// 							"metadata": null,
-	// 							"state": "STATE_NORMAL",
-	// 							"type": "kafka"
-	// 						},
-	// 						"version": "0"
-	// 					},
-	// 					"KafkaBrokers": {
-	// 						"mod_policy": "Admins",
-	// 						"value": {
-	// 							"brokers": [
-	// 								"kafka0:9092",
-	// 								"kafka1:9092"
-	// 							]
-	// 						},
-	// 						"version": "0"
-	// 					}
-	// 				},
-	// 				"version": "1"
-	// 			}
-	// 		},
-	// 		"mod_policy": "",
-	// 		"policies": {
-	// 			"Admins": {
-	// 				"mod_policy": "Admins",
-	// 				"policy": {
-	// 					"type": 3,
-	// 					"value": {
-	// 						"rule": "MAJORITY",
-	// 						"sub_policy": "Admins"
-	// 					}
-	// 				},
-	// 				"version": "0"
-	// 			},
-	// 			"Readers": {
-	// 				"mod_policy": "Admins",
-	// 				"policy": {
-	// 					"type": 3,
-	// 					"value": {
-	// 						"rule": "ANY",
-	// 						"sub_policy": "Readers"
-	// 					}
-	// 				},
-	// 				"version": "0"
-	// 			},
-	// 			"Writers": {
-	// 				"mod_policy": "Admins",
-	// 				"policy": {
-	// 					"type": 3,
-	// 					"value": {
-	// 						"rule": "ANY",
-	// 						"sub_policy": "Writers"
-	// 					}
-	// 				},
-	// 				"version": "0"
-	// 			}
-	// 		},
-	// 		"values": {
-	// 			"Capabilities": {
-	// 				"mod_policy": "Admins",
-	// 				"value": {
-	// 					"capabilities": {
-	// 						"V1_3": {}
-	// 					}
-	// 				},
-	// 				"version": "0"
-	// 			},
-	// 			"OrdererAddresses": {
-	// 				"mod_policy": "Admins",
-	// 				"value": {
-	// 					"addresses": [
-	// 						"127.0.0.1:7050"
-	// 					]
-	// 				},
-	// 				"version": "0"
-	// 			}
-	// 		},
-	// 		"version": "0"
-	// 	},
-	// 	"sequence": "0"
-	// }
-}
-
-func ExampleConfigTx_AddOrdererCapability() {
-	baseConfig := fetchChannelConfig()
-	c := configtx.New(baseConfig)
-
-	err := c.AddOrdererCapability("V1_4")
-	if err != nil {
-		panic(err)
-	}
-}
-
-func ExampleConfigTx_AddApplicationCapability() {
-	baseConfig := fetchChannelConfig()
-	c := configtx.New(baseConfig)
-
-	err := c.AddApplicationCapability("V1_3")
-	if err != nil {
-		panic(err)
-	}
-}
-
-func ExampleConfigTx_RemoveChannelCapability() {
-	baseConfig := fetchChannelConfig()
-	c := configtx.New(baseConfig)
-
-	err := c.RemoveChannelCapability("V1_3")
-	if err != nil {
-		panic(err)
-	}
-}
 
-func ExampleConfigTx_RemoveOrdererCapability() {
-	baseConfig := fetchChannelConfig()
-	c := configtx.New(baseConfig)
-
-	err := c.RemoveOrdererCapability("V1_4")
+	_, err := configtx.NewCreateChannelTx(channel, "testchannel")
 	if err != nil {
 		panic(err)
 	}
 }
 
+// This example shows the addition of a certificate to an application's intermediate
+// certificate list.
 func ExampleConfigTx_SetApplicationMSP() {
```

Comment:
I'm not actually sure how these function names are supposed to be formatted but it does look inconsistent with the rest of them

Reply:
These must be formatted this way because of the ConfigTx type.